utility.py

```diff
@@ -0,0 +1,32 @@
+from discord.ext import commands
+
+
+class Utility:
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command()
+    async def vote(self, ctx: commands.Context, question: commands.clean_content, *options: commands.clean_content):
+        """
+        Start a vote given a question and 2-9 options.
+        Surround the question and options with " if they're more than one word.
+        """
+        if len(options) < 2:
+            raise commands.BadArgument("At least two options are required to create a vote.")
+        if len(options) > 9:
+            raise commands.BadArgument("A vote can have a maximum of 9 options.")
+
+        options_dict = {keycap(i+1): opt for i, opt in enumerate(options)}
+        options_str = '\n'.join(f'{kc} - {opt}' for kc, opt in options_dict.items())
```

Comment:
The `options_dict` and `options_str` line could probably be merged into one to define `options_str` directly, avoiding the need for two loops.